canonical-web-and-design/greenhouse-exporter
# Rebuild static assets on save

## Done
When there's a change in the `scss` or `ts` the static files are build automatically. You still have to refresh the page, though. I couldn't find an easy way to do live reloading.
## QA
- `dotrun --env DEBUG_USER_EMAIL=peter.mahnke@canonical.com`
- http://localhost:8501/
- Change a `tsx` or `scss` file. Save the changes
- Reload the page and check the changes are applied
 ## Issue
#51
